dev/com.ibm.ws.logging.test/build.gradle

```diff
@@ -0,0 +1,10 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
```

Comment:
perhaps add a note to this file indicating why the logging tests need to be in their own project.  Currently we are trying to clean up all of the _test projects by merging them in with their corresponding src projects, so hopefully a well-placed comment would save someone the trouble of running into the same circular dependency issues that you ran into

Reply:
done, and also done for com.ibm.ws.ras.intrument_test project also.